samcli/lib/build/workflow_config.py

```diff
@@ -245,11 +245,7 @@ def _key(c: CONFIG) -> str:
     # A "workflow config" is like a primary key to identify the workflow. So we use the config as a key in the
     # map to identify which workflows can support building within a container.
 
-    unsupported = {
-        _key(DOTNET_CLIPACKAGE_CONFIG): "We do not support building .NET Core Lambda functions within a container. "
-        "Try building without the container. Most .NET Core functions will build "
-        "successfully.",
-    }
+    unsupported: Dict[str, str] = {}
```

Comment:
Why we are keeping empty `unsupported` dict here? It was `dotnet` builders before, but I can't understand what are we setting it now.

Reply:
This makes sense, let's remove if we don't need it.